web/src/OverviewTableColumns.tsx

```diff
@@ -366,7 +366,6 @@ export function TableTriggerColumn({ row }: CellProps<RowValues>) {
       isBuilding={trigger.isBuilding}
       triggerMode={row.values.triggerMode}
       isQueued={trigger.isQueued}
-      resourceName={row.values.name}
```

Comment:
it turns out this was only used to implement `onClick`, but we already have `onTrigger` a few lines down for that purpose, and we were ignoring it!